test/jdk/tools/jpackage/windows/Win8301247Test.java

```diff
@@ -78,75 +68,15 @@ public void test() throws IOException, InterruptedException {
             Thread.sleep(Duration.ofSeconds(10));
 
             // Get PID of the main app launcher process
-            final long pid = findMainAppLauncherPID(cmd, 2).get();
+            final long pid = findAppLauncherPID(cmd, null, 2).get();
 
             // Kill the main app launcher process
-            Executor.of("taskkill", "/F", "/PID", Long.toString(pid)).
-                    dumpOutput(true).execute();
+            killProcess(pid);
```

Comment:
I think this `killProcess` might not be called if `findAppLauncherPID` throws exception when we got only 1 process instead of expected 2.